cardano-api/src/Cardano/Api/InMode.hs

```diff
@@ -282,6 +290,10 @@ deriving instance Show (TxValidationErrorInMode mode)
 
 
 fromConsensusApplyTxErr :: ConsensusBlockForMode mode ~ block
+                        => Consensus.LedgerSupportsProtocol
+                             (Consensus.ShelleyBlock
+                             (TPraos.TPraos Consensus.StandardCrypto)
```

Comment:
Incorrectly fixed at `TPraos`

Reply:
Yep need to fix

Reply:
If we're fixed to the Shelley era then that's fine. Sorry I didn't notice this!